x/slashing/types/keys.go

```diff
@@ -1,12 +1,15 @@
-package slashing
+package types
 
 import (
 	"encoding/binary"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// nolint
```

Comment:
any reason why the linter is disabled here?

Reply:
mostly just because I wanted to combine the const blocks here... I just separated though however, now `nolint` is no longer required and has been removed.